backend/pong/jwt/views/token.py

```
@@ -8,6 +8,7 @@
 
 from jwt import create_access_and_refresh_token
```

Ruff (0.8.2)

`jwt.create_access_and_refresh_token` imported but unused

Remove unused import: `jwt.create_access_and_refresh_token`

(F401)